### PR TITLE
Home page Skeleton

### LINKED ISSUE #74: home page skeleton

### PR DESCRIPTION
**Pull request by josephjquinn**
Added a home page skeleton based on the Figma.

TODO: Update Colors / layout once Figma is updated.

Screenshot 2023-12-02 at 6 31 03 PM

### CODE REVIEW

File: MTM-Frontend/src/pages/HomeLayout/index.tsx [hunk 1]
```diff
@@ -11,6 +11,8 @@ import { TopBar } from "../../components/NavigationBar/TopBar";
 export const HomeLayout: React.FC = () => {
   const [fact, setFact] = useState<string>("");
   const { currentUser } = useAuth();
+  const location = useLocation();
+  const isHomeIndex = location.pathname === "/home";
 
   useEffect(() => {
```
Comment (OliviaLiu417) [inline MTM-Frontend/src/pages/HomeLayout/index.tsx]: Let's get rid of this since this is just an example

Comment (josephjquinn) [reply MTM-Frontend/src/pages/HomeLayout/index.tsx]: what do you mean? like the unnecessary method imports?

Comment (OliviaLiu417) [reply MTM-Frontend/src/pages/HomeLayout/index.tsx]: The useEffect hook to grab the example data

Comment (josephjquinn) [reply MTM-Frontend/src/pages/HomeLayout/index.tsx]: Done

**Review by OliviaLiu417 — APPROVED** [review]

**Pull request merged**